download/index.shtml

```diff
@@ -160,18 +160,16 @@
 <!-- 	</dd> -->
 
 	<dt class="im">
-<a href="http://java.com/java/download/index.jsp?cid=jdp74834" target="_blank">
-<img width="88" height="31" border="0" style="margin: 8px;" alt="GetJava Download Button" title="GetJava" src="http://java.com/en/img/everywhere/getjava_sm.gif?cid=jdp74834">
-</a>
-    <dt><a href="http://java.com/java/download/index.jsp?cid=jdp74834">Java</a></dt>
+        <a href="http://www.java.com" target="_blank"><img src="http://download.oracle.com/technetwork/java/get-java/getjavasoftware-88x31.png" alt="Get Java Software" border="0" width="88" height="31" /></a>
```

Comment:
In fact this was the url that Oracle used behind its button. Changed it, as that requires the user to look for the actual download button/page after being sent to the Oracle web site.